DetectDynamicJS.py

```diff
@@ -77,6 +77,11 @@ def doPassiveScan(self, baseRequestResponse):
         statusCode = responseInfo.getStatusCode()
         if statusCode != 200:
           return
+        
+        # Check if the script is world readable
```

Comment:
I am. Just give me a few weeks. I'm not full-time working on this, will have to find some free time first.